prometheus/generic/add-target/templates/prometheus_target.yml.j2

```diff
@@ -0,0 +1,6 @@
+- targets:
+{% if hostvars[item].datasource_url.startswith('https://') %}
+  - {{ hostvars[item].datasource_url[8:] }}
+{% else %}
+  - {{ hostvars[item].datasource_url }}
+{% endif %}
```

Comment:
Do we need to account for `http://` as well?

Reply:
The datasource url is usually https://prometheus-k8s-monitoring.domain.com, so this fits our use case. To support http:// we would need to create some logic in the prometheus.yml, because this template creates targets for job:

- job_name: 'federated_prometheus'
    scrape_interval: 5s
    scheme: 'https'

Scheme says https.